example_dags/example_dynamic_task_template.py

```diff
@@ -0,0 +1,37 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+
+from astro import sql as aql
+from astro.files import get_file_list
+from astro.sql.operators.load_file import LoadFile
+from astro.sql.table import Metadata, Table
+
+GCS_BUCKET = os.getenv("GCS_BUCKET", "gs://dag-authoring/dynamic_task/")
+ASTRO_GCP_CONN_ID = os.getenv("ASTRO_GCP_CONN_ID", "google_cloud_default")
+ASTRO_BIGQUERY_DATASET = os.getenv("ASTRO_BIGQUERY_DATASET", "dag_authoring")
+
+
+# [START howto_operator_get_file_list]
+
+
+with DAG(
+    dag_id="example_dynamic_task_template",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    catchup=False,
+) as dag:
+    LoadFile.partial(
```

Comment:
@kaxil here ```LoadFile``` is a like traditional operator.  yesterday I was discussing with @tatiana to make make the traditional private in this repo. What is your thought on this?

Reply:
Only if we then have a non "private" replacement for it in examples. We don't want to advocate for calling a private operator, do we?

Reply:
Yeah like Felix mentioned - we can't prefix the `LoadFile` with `_` because we need it for Dynamic Task Mapping to work. Dynamic Task Mapping works with "an Operator" class (not instance of the operator) not "task" whereas `load_file` returns an XComArg (output of task) - (`.output` on instantiated operator)